src/web/components/StickyBottomBanner/BrazeBanner.tsx

```diff
@@ -159,41 +158,30 @@ export const BrazeBanner = ({ meta }: Props) => {
 	>();
 
 	useEffect(() => {
-		if (meta) {
-			// TODO: unify the way we handle sharing these deps (this is
-			// duplicated in SlotBodyEnd). Probably via the automat client
-			// library.
-			window.guardian.automat = {
-				react: React,
-				preact: React,
-				emotionCore,
-				emotionTheming,
-				emotion,
-			};
-
-			import(
-				/* webpackChunkName: "guardian-braze-components" */ '@guardian/braze-components'
-			)
-				.then((module) => {
-					setBrazeComponent(() => module.BrazeMessage);
-				})
-				.catch((error) =>
-					window.guardian.modules.sentry.reportError(
-						error,
-						'braze-banner',
-					),
-				);
-		}
-	}, [meta]);
```

Comment:
`meta` is never used in the `useEffect` and according to the componet prop it is always defined

Reply:
Huh yep you're right. I wonder why the condition exists, maybe in a previous version of this it was needed 🤔   Since we've now got a `useEffect` with an empty dependency array, causing it to only run once, perhaps `useOnce` would be better?